tests/ballerina-integration-test/src/test/resources/http/httpservices/100_continue.bal

```diff
@@ -38,4 +39,22 @@ service<http:Service> helloContinue bind { port: 9090 } {
             }
         }
     }
+
+    @http:ResourceConfig {
+        methods: ["POST"]
+    }
+    getFormParam(endpoint caller, http:Request req) {
+        string replyMsg = "Result =";
+        mime:Entity[] bodyParts = check req.getBodyParts();
+        int i = 0;
+        while (i < lengthof bodyParts) {
+            mime:Entity part = bodyParts[i];
+            mime:ContentDisposition contentDisposition = part.getContentDisposition();
+            replyMsg += " Key:" + contentDisposition.name + " Value: " + check part.getBodyAsString();
```

Comment:
Better if we could use a `StringBuilder` inside the while loop.

Reply:
Does ballerina have a `StringBuilder`? :open_mouth:

Reply:
AFAIK, we don't have.

Reply:
Yeah, I also not aware of come feature.

Reply:
Ops, didn't see that this is a .bal file. Let's stick into this. Anyway it is nice to have such a feature ;)